ppr-ui/src/components/common/InfoChip.vue

```diff
@@ -7,7 +7,7 @@
     :textColor="chipColors.textColor"
     :data-test-id="`${action}-badge`"
   >
-    <b>{{ action }}</b>
+    <v-icon class="mr-1">mdi-lock</v-icon><b>{{ action }}</b>
```

Comment:
Does the lock icon require a conditonal? 
This will add it to all the badges.

PR link:
![image](https://github.com/bcgov/ppr/assets/77707952/4e446310-d6a3-47dc-8bfb-8f73f8367888)

![image](https://github.com/bcgov/ppr/assets/77707952/aa249578-614a-4088-bc21-3adaab8cf966)


Dev:
![image](https://github.com/bcgov/ppr/assets/77707952/94b04724-f044-4122-9eab-0a9e58ddc0bb)

Reply:
Ohh good catch, there is supposed to be a condition. Will add right away!